src/App.tsx

```diff
@@ -20,17 +32,19 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={handleChange}
               />
             </div>
           </div>
         </div>
 
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList movies={visibleMovies} />
       </div>
 
       <div className="sidebar">
         Sidebar goes here
       </div>
     </div>
   );
-};
+};
```

Comment:
add empty line after the code
```suggestion
};

```